modules/core/src/lib/attribute-manager.js

```diff
@@ -282,8 +282,13 @@ export default class AttributeManager {
     for (const attributeName in attributes) {
       const attribute = attributes[attributeName];
       if (attribute.needsRedraw({clearChangedFlags: true})) {
-        // Only return non-transition attributes
-        if (!attributeTransitionManager.hasAttribute(attributeName)) {
+        if (attribute.userData.shaderAttributes) {
```

Comment:
The `changedAttributes` from `TransitionManager` could be interleaved as well. We need to loop through all changed attributes and check if they have `shaderAttributes`.